crm_update_sales_team_stages/wizard/crm_team_stage_wizard.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+from odoo import fields, models, _
+
+
+class CrmTeamStageWizard(models.TransientModel):
+    _name = "crm.team.stage.wizard"
+
+    state = fields.Selection(
+        selection=[
+            ('start', 'Start'),
+            ('done', 'Done')
+        ],
+        default='start'
+    )
+
+    name = fields.Char(
```

Comment:
@wardm95 waarom geen One2many of e.d zodat je de verschillende bestaande fase namen wel kan zien in de wizard? Nu is het kinda gokken 😓 Bvb "In Progress" versus "In progress".